keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -220,11 +229,12 @@ All behavior change is encapsulated within the Kubelet, so there is no version s
 
 ###### How can this feature be enabled / disabled in a live cluster?
 
-- [x] Feature gate
-  - Feature gate name: KubeletDropInConfig
-  - Components depending on the feature gate: Kubelet
+- [] Feature gate
+N/A
+
+In addition to configuring the KUBELET_CONFIG_DROPIN_DIR_ALPHA environment variable, administrators must explicitly set the --config-dir flag in the kubelet's command-line interface (CLI) to enable this feature.. Starting from the beta phase, specifying the --config-dir flag is the only way to enable this feature. The default value for --config-dir is an empty string, which means the feature is disabled by default.
```

Comment:
ok, I can live with that. it still doesn't address one thing - the fact that people pull example commands from the web, and if those commands include the `--config-dir` flag, the users of those will not actually be acknowledging that they know they are using a beta feature.

Reply:
How about we add a note about that in the example?

Reply:
Something that was done [elsewhere](https://github.com/kubernetes/component-base/blob/7e9e5184583ae274508a6241865298351cdb53df/logs/api/v1/kube_features.go#L37-L52) is to have alpha/beta feature gates. It's an error to use an alpha feature unless the alpha feature gate is on (which it isn't by default). Same for beta, except that it is on by default. Cautious users can turn it off to ensure that they don't accidentally use beta features.

Not sure how applicable that could be here, just my two cents... :smile: